src/test/java/net/snowflake/ingest/streaming/internal/datatypes/AbstractDataTypeTest.java

```diff
@@ -112,6 +122,21 @@ protected String createTable(String dataType) throws SQLException {
             String.format(
                 "create or replace table %s (%s string, %s %s)",
                 tableName, SOURCE_COLUMN_NAME, VALUE_COLUMN_NAME, dataType));
+
+    return tableName;
+  }
+
+  protected String createIcebergTable(String dataType) throws SQLException {
+    String tableName = getRandomIdentifier();
+    conn.createStatement()
+        .execute(
+            String.format(
+                "create or replace iceberg table %s (%s string, %s %s) "
+                    + "catalog = 'SNOWFLAKE' "
+                    + "external_volume = 'streaming_ingest' "
+                    + "base_location = 'all_type_col';",
```

Comment:
1. lets use a different base_location for each table.
2. Create a JIRA for how to cleanup all of the data that ITs upload, from the external volume.

Reply:
1. Set up different base_location for each tests.
2. [JIRA](https://snowflakecomputing.atlassian.net/browse/SNOW-1727545) created.